src/sentry/api/endpoints/organization_events_root_cause_analysis.py

```diff
@@ -21,16 +85,30 @@ def get(self, request, organization):
         ):
             return Response(status=404)
 
-        root_cause_results = {}
-
+        # TODO: Extract this into a custom serializer to handle validation
         transaction_name = request.GET.get("transaction")
         project_id = request.GET.get("project")
-        if not transaction_name or not project_id:
+        regression_breakpoint = request.GET.get("breakpoint")
+        request_start = request.GET.get("requestStart")
+        request_end = request.GET.get("requestEnd")
```

Comment:
@Zylphrex Good point, I wanted to decouple the start and end for this request because I didn't want to conflict with the UI's start, end, statsPeriod settings in the page filters

I can just make sure the frontend request is sending the proper `start` and `end` parameters because the issue details UI is supposed to show the detection range and not adhere to the page filters